eng/scripts/vs.16.buildtools.json

```diff
@@ -14,6 +14,8 @@
         "Microsoft.VisualStudio.Component.NuGet.BuildTools",
         "Microsoft.VisualStudio.Component.VC.ATL",
         "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.ATL.ARM64",
+        "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
```

Comment:
I'd rather we deal w/ the developer experience side of this separately. Please put the eng/scripts/vs*.json changes into a separate PR.

Reply:
So do you want me to do this in phases?  I can get this first PR in where it just builds arm64 binaries, a second PR which deals with installer builds, and then a 3rd PR where we actually test things (somehow), and a final cleanup PR and stuff like the VS install.  But I was planning on just making one mega PR and leaving this in draft for ages.

Reply:
Yeah, your proposed breakdown makes sense. Doing it in pieces might mean a few bits need to be cleaned up in a later build but will make the reviews significantly easier.